### DISCUSSION TITLE
Bike keeps pedalling

### DISCUSSION THREAD

**Discussion by debaashaas11** [danielnilsson9/bbs-fw#221] [General]
Hello, Thanks for opening this open to the public. I currently started the bike and it starts pedaling without any human input. Do you know what settings I need to change to only start the motor if I start pedaling.

**Comment by WvBijsterveld** [comment]
change the throttle start input to 1.1v, if it keeps happening raise it until it doesn’t happen anymore.

**Reply by debaashaas11** [reply]
Super thank you! I will try it out tonight and let you know

I tried version V1.4 of the firmware but thereafter i installer version V1.3 can I safely go back to V1.4?

**Reply by dav0000000** [reply]
Was the problem also in v1.3?

**Reply by debaashaas11** [reply]
Yes, I hoped a other version would resolve this. But I will try the 1.1v fix.

I just hope i can upgrade it back to version 1.4

**Reply by dav0000000** [reply]
The 1.1 volt throttle start should fix it. I was just wondering if it somehow worked on v1.3 because it should be the same problem there. Thanks for confirming that.

**Reply by debaashaas11** [reply]
Do you also know if i can flash V1.4 back on my bike?

**Reply by dav0000000** [reply]
Yes you can just load v1.4. After you load v1.4 just use the v1.4 app to change the values you need.

**Reply by debaashaas11** [reply]
I @dav0000000 @WvBijsterveld 
Also another thing that I noticed that my screen doesnt register the power going to the motor, and it also has the wrong battery reading. Is there a way to fix those?

**Comment by debaashaas11** [comment]
Super thank you! I will try it out tonight and let you know.

Best ,

Ian

Op di 21 mei 2024 08:32 schreef Wesley ***@***.***>:

**Comment by WvBijsterveld** [comment]
Yea.

**Comment by WvBijsterveld** [comment]
Yes you can, should be no different, if any issue begins check your configuration. otherwise use the latest firmware u have no problems with.

**Comment by WvBijsterveld** [comment]
with power output reading i can’t help, with battery SOC reading, so the %. it’s to do with the upper V limit and low volt limit. LVC and HVC i believe. simply just enter on the high one how many volts your battery is when full and on the low when it is completely empty. that should do the trick

**Reply by debaashaas11** [reply]
@WvBijsterveld @dav0000000 the 1.1v trick worked! Many thanks!!! It cuts off when I fully press the throttle fully down so i need to adjust the upper voltage.

For the battery voltage limit do i need a multimeter or just read the specs of the battery?

**Comment by WvBijsterveld** [comment]
just read the specs of your battery 
, you can search the voltage cutoffs for 48/52v and enter them accordingly.

**Reply by debaashaas11** [reply]
I have a 48v 20ah hailong max battery

Would that mean 42v cutoff?

54.6 high cutoff?

**Comment by WvBijsterveld** [comment]
54.6 is perfect, the 42v limit sounds right but there is a subjective range. depending on if you want more cycles out of your battery(it to last longer) by not going too low charge for the battery.
but  you should look into that if interested

to sum it up the values you have given are perfect
